crates/frontend/src/components/experiment.rs

```diff
@@ -162,62 +187,52 @@ where
                             for token in contexts.clone() {
                                 let (dimension, values) = (token.left_operand, token.right_operand);
                                 let mut value_views = Vec::new();
-
                                 for value in values.iter() {
                                     if value.is_object() && value.get("var").is_some() {
                                         continue;
                                     }
-
                                     let value_str = match value {
```

Comment:
can this not be replaced by `html_display` ?

Reply:
Good point, will do it